spec/definitions/DataExport.yaml

```diff
@@ -47,13 +47,45 @@ properties:
       criteria:
         $ref: '#/definitions/Condition'
   dateRange:
-    $ref: '#/definitions/DateRange'
+    type: object
+    description: The date range (can be in relative formats).  If omitted, all time will be included.
+    properties:
+      start:
+        type: string
+        description: Any valid datetime arguments including [relative datetime arguments](http://php.net/manual/en/datetime.formats.php).
+        example: 'yesterday'
+      end:
+        type: string
+        description: Any valid datetime arguments including [relative datetime arguments](http://php.net/manual/en/datetime.formats.php).
+        example: 'today'
+    required:
+      - start
+      - end
   emailNotification:
     description: List of emails to be notified when export is completed
     type: array
     items:
       type: string
       format: email
+  fields:
+    description: List of fields to include in the export.  See supporting documentation for more info.  If omitted, default fields will be included.
+    type: array
+    items:
+      type: string
+  recurring:
```

Comment:
Can this be put into a separate PR since scheduling is not working right now?

Reply:
We want to provide final documentation.  So it's okay if it doesn't work at the moment.